### ISSUE TITLE
Many-to-one relation does not work

### ISSUE THREAD

**Issue by c000** [kamichidu/goen#9]
Many-to-one relation with multiple column does not work.

Work
```go
type Table struct {
	ignore some fields...
	ColOne   int
	ColTwo   int
	RefTable *RefTable `foreign_key:"col_one"`
}
```

NOT Work
```go
type Table struct {
	ignore some fields...
	ColOne   int
	ColTwo   int
	RefTable *RefTable `foreign_key:"col_one,col_two"`
}
```

**Comment by kamichidu** [comment]
thanks for your reporting!
i'll check it lator.

**Comment by kamichidu** [comment]
reproduced under my environment.
i'll investigate ASAP.

**Comment by kamichidu** [comment]
fixed version v0.0.10 was pushed.
update goen and re-generate, please.
